NSS-Day-Cohort-49/react-nutshell-state-slingers
# Mat friends

#### Changes Made
1. Added files `FriendsProvider, FriendList, FriendCard, FriendForm` to `friend` directory.
1. Modified file `ApplicationViews` to include friend route.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout mat-friends
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Navigate to friends page. It should display friends of logged in user.
    > When a user's full name (whitespace and case should not matter) or username is entered, a friend relationship should be created and immediately appear.
    > A friend relationship that already exists or a name that does not find a match should generate an alert
    > When a user clicks the delete button, the related relationship should be deleted and the friend should be removed from the list
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
